cw-relayer/relayer/client/tx.go

```diff
@@ -22,14 +20,12 @@ func BroadcastTx(clientCtx client.Context, txf tx.Factory, msgs ...sdk.Msg) (*sd
 	}
 
 	//error for now: alternative set high gas limit
```

Comment:
Are we going with the alternative for now and just setting a high  gas limit? Did you run into issues with not having enough gas using the `CalculateGas` function?

Reply:
`CalculateGas` function returns an error due to a scrt network dependency

Reply:
Could help to add that as a comment. This is the client we are using for all the chains right? Is this how we want to handle all of them?

Reply:
I have added a comment, this happens only specifically to scrt network so it might not be necessary to have this on other cosmwasm supported chains.